pygdf/dataframe.py

```diff
@@ -852,10 +852,14 @@ def groupby(self, by, sort=False, as_index=False):
         sort : bool
             Force sorting group keys.
             Depends on the underlying algorithm.
-            Current algorithm always sort.
         as_index : bool; defaults to False
             Must be False.  Provided to be API compatible with pandas.
             The keys are always left as regular columns in the result.
+        method: str, optional
```

Comment:
nitpick, but for numpydoc, there needs to be a space on both sides of the colon, as above